src/etc/inc/services.inc

```diff
@@ -113,7 +113,9 @@ function services_radvd_configure($blacklist = array())
 
     /* handle manually configured DHCP6 server settings first */
     foreach ($config['dhcpdv6'] as $dhcpv6if => $dhcpv6ifconf) {
-        if (!isset($config['interfaces'][$dhcpv6if]['enable'])) {
+        if (isset($config['interfaces'][$dhcpv6if]['track6-interface'])) {
+            continue;
+        } else if (!isset($config['interfaces'][$dhcpv6if]['enable'])) {
```

Comment:
elseif as per PSR style :)

overall this looks like a leftover of manual/static configuration, but it should go in as there is no way to clean this up from the GUI at the moment.